KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -430,4 +442,38 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
     func getLoyaltyNonce(completion: @escaping (Result<LoyaltyNonce>) -> Void) {
         return loyaltyView.getLoyaltyPreAuthNonce(completion: completion)
     }
+
+    func quoteDidExpire() {
+        let expiredAlertController = UIAlertController(
```

Comment:
Use `AlertHandler` to show alerts. It has some logic to force light mode until we introduce dark mode to the app. Otherwise your alert would be dark on some phones while the rest of the UI would be light.

Reply:
Great, thanks for this suggestion :)